react-11/src/GuessingNumberGame.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import _ from 'lodash';
+
+export default class GuessingNumberGame extends React.Component {
+  constructor(props) {
+      super(props);
+        this.state = {
+            ans: this.getRandomNumber(1, 1000),
+            guess: 0,
+        }
+  }
+
+  getRandomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  showMsg(guess, ans) {
+      if (guess) {
+      const showCorrentAns = Math.random() > 0.2;
+      if (showCorrentAns) {
+          if (guess === ans) {
+              return <p>Correct !</p>
+          }
+        return guess < ans ? <p>Your guess is too low</p> : <p>Your guess is too high</p>
```

Comment:
Never return JSX/HTML elements this way.
Only the content should be returned, all the element conditions should be in the `return/render` block and the actual content should be set in the state.